vNext/src/OrleansProviders/OrleansProviders.csproj

```diff
@@ -0,0 +1,199 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform>AnyCPU</Platform>
+    <ProductVersion>10.0.20506</ProductVersion>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{A054DB14-2A92-4CC0-959E-A2C51F5E65DA}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Orleans.Providers</RootNamespace>
+    <AssemblyName>OrleansProviders</AssemblyName>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Orleans.Providers should be targetting .NET Standard, as it is not a test project.